pom.xml

```diff
@@ -72,43 +68,40 @@
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
-    <maven-hpi-plugin.version>1.96</maven-hpi-plugin.version>
+    <jenkins.version>1.609.3</jenkins.version>
```

Comment:
Why the large bump in version? Do not recommend going past 1.580.1 unless there is a specific reason.